backend/Domain/Entities/Food.cs

```diff
@@ -8,13 +8,16 @@ public class Food : BaseEntity
         public decimal StartingPrice { get; set; }
         public DateTime CreatedAt { get; set; }
         public string IdRestaurant { get; set; }
+        public string ImageURL { get; set; }
         public virtual IEnumerable<TypeOfFood> TypesOfFood { get; set; }
         public DateTime StartDecreasingAt { get; set; }
         public DecreaseType DecreaseType { get; set; }
         public double IntervalTimeInMinutes { get; set; }
         public decimal AmountPerInterval { get; set; }
         public double PercentPerInterval { get; set; }
 
+        private const string DEFAULT_IMAGE_URL = "https://genesisairway.com/wp-content/uploads/2019/05/no-image.jpg";
```

Comment:
this same URL is used in 3 files, I believe all of them should import from one constant